src/jit/lsra.cpp

```diff
@@ -6010,21 +6266,51 @@ regNumber LinearScan::allocateBusyReg(Interval* current, RefPosition* refPositio
         assert(farthestRefPhysRegRecord != nullptr);
         if ((farthestLocation == refLocation) && !refPosition->isFixedRegRef)
         {
+#ifdef _TARGET_ARM_
+            Interval* assignedInterval =
+                (farthestRefPhysRegRecord == nullptr) ? nullptr : farthestRefPhysRegRecord->assignedInterval;
+            Interval* assignedInterval2 =
+                (farthestRefPhysRegRecord2 == nullptr) ? nullptr : farthestRefPhysRegRecord2->assignedInterval;
+            RefPosition* nextRefPosition =
+                (assignedInterval == nullptr) ? nullptr : assignedInterval->getNextRefPosition();
+            RefPosition* nextRefPosition2 =
+                (assignedInterval2 == nullptr) ? nullptr : assignedInterval2->getNextRefPosition();
+            if (nextRefPosition != nullptr)
```

Comment:
I almost forgot that we need braces when there is a `else`  even when there is a single line statement. (
https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md#121-braces-for-if)

Thanks :)